vizro-core/src/vizro/__init__.py

```diff
@@ -13,7 +13,7 @@
 
 __all__ = ["Vizro"]
 
-__version__ = "0.1.21.dev0"
+__version__ = "0.1.21"
```

Comment:
Don't we need to bump the version in our mkdocs.yml also now? We were planning to add a point to the release checklist to do that.